osf/models/brand.py

```diff
@@ -15,3 +17,6 @@ class Brand(BaseModel):
 
     primary_color = models.CharField(max_length=7)
     secondary_color = models.CharField(max_length=7)
+
+    def get_absolute_url(self):
+        return api_v2_url('brands/{}/'.format(self.id))
```

Comment:
This should probably be a reverse so we don't have to update it with the endpoint like:
```
return absolute_reverse(
            'brands:brands-detail', kwargs={
                'provider_id': self.context['request'].parser_context['kwargs']['version'],
                'moderator_id': obj._id,
                'version': self.context['request'].parser_context['kwargs']['version'],
            },
        )
```
Something like that.